src/map/latent_effect_container.cpp

```diff
@@ -813,6 +814,14 @@ bool CLatentEffectContainer::ProcessLatentEffect(CLatentEffect& latentEffect)
                         break;
                     }
                 }
+                for (auto trust : static_cast<CCharEntity*>(member)->PTrusts)
+                {
+                    if (trust->GetMJob() == latentEffect.GetConditionsValue())
```

Comment:
It was on Topaz, because I `!additem`'d myself a `Forte Earring`. I don't have confirmation that Trusts trigger Latents, but (the wikis say) that they affect COR Rolls, so that was my logical jump